index.html

```diff
@@ -1,14 +1,244 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Homework 6</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="Styles/style.css">
 </head>
+
 <body>
+    <div class="main">
+        <div class="header ">
+            <div class="navigation">
+                <div class="site-logo"><img src="Images/KIDS-jump-TECH-2 2.png" alt=""></div>
+                <div class="nav-menu">
+                    <a href="">PRODUCT</a>
+                    <a href="">TEST DRIVE</a>
+                    <a href="">FOR KIDS</a>
+                    <a href="">GAMES</a>
+                    <a href="">CONTACTS US</a>
+                    <a href="">FAQ</a>
+                </div>
+
+            </div>
+            <div class="header-section">
+                <div class="txt-section">
+                    <div class="about txt-white">GAMES</div>
+                    <div class="about-mini txt-white">More than 90 educational games and applications are included with
+                        the delivery of the interactive floor.</div>
+                </div>
+                <div class="icon-section">
+                    <div class="icon1"><img src="Images/icon1.png" alt=""></div>
```

Comment:
not good name for class icon1, icon2, etc

Reply:
changed